docs/fenix-custom-targeting.md

```diff
@@ -0,0 +1,33 @@
+---
+id: fenix-custom-targeting
+title: Adding new targeting parameters to Fenix
+---
+# Adding new targeting parameters to Fenix
+This page demonstrates how to add new targeting parameters to Fenix, enabling experiment creators more specific targeting.
+For more general documentation on targeting custom audiences, check out [the custom audiences docs](console/custom-audiences.md)
+
+## Adding the attribute to the application
+The Nimbus SDK exposes a new `customTargetingAttributes` parameter in its initializer that is a `Map<String, String>` map. We can take advantage of this parameter to pass in new targeting attributes without modifying the Nimbus SDK at all.
+:::warning
+A current limitation is that both the key and the value of the targeting parameter are **strings**. Please reach out to the Nimbus SDK team for any targeting parameters that require integer comparison, or any other richer `JEXL` expressions that cannot be done with strings.
+:::
+
+
+### How to add a new attribute
+In [NimbusSetup.kt](https://github.com/travis79/fenix/blob/sdk-344-first-run-flag/app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt#L67) the map `customTargetingAttributes` will be used to add custom targeting. Simply add a new key-value pair to the map and it will be available for targeting. For example:
```

Comment:
Reminder to myself to update this to point to main once the PR is merged (do the same for ios)